app-misc/remind/remind-04.00.02.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Ridiculously functional reminder program"
+HOMEPAGE="https://dianne.skoll.ca/projects/remind/"
+SRC_URI="https://dianne.skoll.ca/projects/remind/download/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="tk"
+
+RDEPEND="
+	tk? ( >=dev-lang/tk-8.5 dev-tcltk/tcllib )
+"
+DEPEND="${RDEPEND}
+	dev-perl/Cairo
+	dev-perl/JSON-MaybeXS
+	dev-perl/Pango
+	virtual/perl-Getopt-Long
+"
+DOCS="docs/* examples/defs.rem "
+
+src_prepare() {
+	default
+	sed -i 's:$(MAKE) -C src install:&-nostripped:' "${S}"/Makefile || die
+}
+
+src_test() {
+	if [[ ${EUID} -eq 0 ]] ; then
+		ewarn "Testing fails if run as root. Skipping tests"
+	else
+		emake test
+	fi
+}
+
+src_install() {
+	default
+
+	if ! use tk ; then
+		rm \
+			"${D}"/usr/bin/cm2rem* \
+			"${D}"/usr/bin/tkremind \
+			"${D}"/usr/share/man/man1/cm2rem* \
+			"${D}"/usr/share/man/man1/tkremind* \
+			|| die
```

Comment:
```
Installing /var/tmp/portage/app-misc/remind-04.00.02/image/usr/share/man/man1/rem2pdf.1
Installing /var/tmp/portage/app-misc/remind-04.00.02/image/usr/share/man/man3/Remind::PDF.3pm
Installing /var/tmp/portage/app-misc/remind-04.00.02/image/usr/share/man/man3/Remind::PDF::Entry.3pm
Installing /var/tmp/portage/app-misc/remind-04.00.02/image/usr/bin/rem2pdf
make[2]: Leaving directory '/var/tmp/portage/app-misc/remind-04.00.02/work/remind-04.00.02/rem2pdf'
make[1]: Leaving directory '/var/tmp/portage/app-misc/remind-04.00.02/work/remind-04.00.02/rem2pdf'
rm: cannot remove '/var/tmp/portage/app-misc/remind-04.00.02/image/usr/bin/cm2rem*': No such file or directory
rm: cannot remove '/var/tmp/portage/app-misc/remind-04.00.02/image/usr/share/man/man1/cm2rem*': No such file or directory
 * ERROR: app-misc/remind-04.00.02::gentoo failed (install phase):
 *   (no error message)
 * 
 * Call stack:
 *     ebuild.sh, line 122:  Called src_install
 *   environment, line 529:  Called die
 * The specific snippet of code:
 *           rm "${D}"/usr/bin/cm2rem* "${D}"/usr/bin/tkremind "${D}"/usr/share/man/man1/cm2rem* "${D}"/usr/share/man/man1/tkremind* || die;
```
Wonder if the man pages are somehow automatically generated if you have the right dep installed?

Reply:
I'll check this. The release history did not mention dependencies for the man page, so it's probably implied. I'll also fix the ebuild to install the vim syntax file.

Reply:
Vim file installation fixed, but I still need to figure out the rem2pdf error, please do not merge yet.

Reply:
cm2rem issue resolved, ready to merge.